semantics/common/configuration.k

```diff
@@ -101,6 +101,18 @@ module COMMON-CONFIGURATION
                          <access-specifier> .K </access-specifier>
                     </class>
                </classes>
+
+               <cpp-enums>
+                    <cppenum multiplicity="*" type="Map">
+                         <enum-id> 0 </enum-id>
```

Comment:
But what about unnamed enums (and classes)? We are able to decltype() an instance of anonymous enum/class.